### ISSUE TITLE
Nametags can't be hidden

### ISSUE THREAD

**Issue by MClausz** [CodeCrafter47/BungeeTabListPlus#507]
I tried to hide a player's nametag by using team.setOption(Option.NAME_TAG_VISIBILITY, OptionStatus.NEVER); but their nametag still be shown when BungeeTabListPlus 3.0.x is installed.
This bug did not happen with BungeeTabListPlus 2.8.12

**Comment by CodeCrafter47** [comment]
What is the size of the tab list. Is it using 80 slots or less? Which version of the minecraft client are you using?

**Comment by MClausz** [comment]
The tablist size is 80, version is 1.12.2 (client & server) and I think it happens to all client version
Correction: the nametag is actually hidden but is shown again after a moment for some reason
I will record a video soon

**Comment by CodeCrafter47** [comment]
Can you check if the latest dev-build fixes the issue?

**Comment by MClausz** [comment]
No, somehow their nametag is show again and the prefix, suffix is lost
https://i.imgur.com/vkPc4hU.png

**Comment by MClausz** [comment]
I just discovered that this issue is caused by GlowAPI (https://www.spigotmc.org/resources/api-glowapi.19422/), I'm using version 1.4.6 because 1.4.7+ only support MC 1.13+
When I make player A to be glowing to player B using GlowAPI, A's nametag will be shown to B, without suffix and prefix.

**Comment by CodeCrafter47** [comment]
So this issue can be closed then?

**Comment by MClausz** [comment]
I really hope you can fix this as this issue doesn't happen with older version of BungeeTabListPlus 
If not, please close this issue

**Comment by CodeCrafter47** [comment]
So you are saying when using GlowAPI, but not BungeeTabListPlus, then there is no issue?

**Comment by MClausz** [comment]
Yes, BungeeTabListPlus version 2.8.12 and earlier doesn't have this issue

**Comment by CodeCrafter47** [comment]
I'm not asking about older versions, I'm asking if the issue is present without BungeeTabListPlus.

**Comment by MClausz** [comment]
There is no such issue without BungeeTabListPlus

**Comment by CodeCrafter47** [comment]
Can you make a small test plugin that can trigger the issue?

**Comment by MClausz** [comment]
Sorry for late reply, I'm afraid to make a test plugin because it's a bit complicated, I will make it when I have time.

**Comment by CodeCrafter47** [comment]
When testing last week I made a small plugin to use GlowAPI with commands:
```yaml
        getCommand("glow").setExecutor((sender, command, label, args) -> {

                    //Set the event's player glowing in DARK_AQUA for all online players
                    GlowAPI.setGlowing(((Player) sender), GlowAPI.Color.DARK_AQUA, Bukkit.getOnlinePlayers());
                    return true;
        });
        getCommand("unglow").setExecutor((sender, command, label, args) -> {
            GlowAPI.setGlowing(((Player) sender), GlowAPI.Color.NONE, Bukkit.getOnlinePlayers());
            return true;
        });
```
Then I used Minecrafts builtin scoreboard command to modify the players team. It appeared as though the commands were just undoing the others effect. I could not find any difference between using BTLP and not using it.
